doc/source/whatsnew/v0.23.0.txt

```diff
@@ -419,6 +419,7 @@ Datetimelike
 - Bug in ``.astype()`` to non-ns timedelta units would hold the incorrect dtype (:issue:`19176`, :issue:`19223`, :issue:`12425`)
 - Bug in subtracting :class:`Series` from ``NaT`` incorrectly returning ``NaT`` (:issue:`19158`)
 - Bug in :func:`Series.truncate` which raises ``TypeError`` with a monotonic ``PeriodIndex`` (:issue:`17717`)
+- Bug :func:`Timedelta.__mod__`, :func:`Timedelta.__divmod__` where operating with timedelta-like or numeric arguments would incorrectly raise ``TypeError`` (:issue:`19365`)
```

Comment:
can you make this in to a sub-section and give a couple of examples. also add a small example in the timedelta docs.

Reply:
put it in api changes section

Reply:
> can you make this in to a sub-section and give a couple of examples.

Sure.  For the ipython code blocks, do I copy/paste the traceback?